Packages/com.unity.multiplayer.samples.coop/Utilities/Net/UnityRelayUtilities.cs

```diff
@@ -39,7 +40,8 @@ public static async
                 throw new Exception($"Creating join code request has failed: \n {exception.Message}");
             }
 
-            return (allocation.RelayServer.IpV4, (ushort)allocation.RelayServer.Port, allocation.AllocationIdBytes,
+            var dtlsEndpoint = allocation.ServerEndpoints.First(e => e.Secure);
```

Comment:
🚫 instead of finding `e.Secure`, compare against `"dtls"`. The `e.Secure` might be enabled when/if Relay supports TLS (secured TCP), and this code would break.

Reply:
[680588e](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/pull/485/commits/680588e09f045f797ed8706a19d8e6bf1038dda1)